doc/commands/rcheckin.md

```diff
@@ -11,24 +11,55 @@ rcheckin differs from [checkin](checkin.md) in that the latter squashes the comm
 ## Synopsis
 
     Usage: git-tfs rcheckin [options]
-    where options are:
```

Comment:
Thanks a lot!
Could you keep this line?

Reply:
Sure, but I copied straight out of the command line output.  I think this synopsis and the help text should be auto-generated from a common source, somehow.